README.md

```diff
@@ -122,6 +122,8 @@ frameworks.
 | [Astro] ([Demo][demo29])                                              | An Astro example using `graphql-request` to fetch data.                                                                  |
 | [Vanilla JS] ([Demo][demo30])                                         | Query in the browser with no frameworks, just use the Fetch API.                                                         |
 | [NextAuth] ([Demo][demo31])                                           | Authenticate with NextAuth.js and update account info with GraphCMS.                                                     |
+| [Swift] ([Demo](with-swift))                                                               | A native Swift (iOS & Mac) example.                                                     |
```

Comment:
Typically the `demo` is something the user can click on to try out straight away. Is there a hosted service that we could use for something similar?